package.json

```diff
@@ -39,40 +39,40 @@
     "pages": "npm run storybook:build && cp -r storybook-static _site"
   },
   "dependencies": {
-    "@uswds/uswds": "^3.8.2",
-    "lit": "^3.2.0",
-    "sass": "^1.78.0"
+    "@uswds/uswds": "^3.9.0",
+    "lit": "^3.2.1",
+    "sass": "^1.79.4"
   },
   "devDependencies": {
     "@chromatic-com/storybook": "^1.8.0",
     "@custom-elements-manifest/analyzer": "^0.10.3",
-    "@storybook/addon-a11y": "^8.2.9",
-    "@storybook/addon-docs": "^8.2.9",
-    "@storybook/addon-essentials": "^8.2.9",
-    "@storybook/addon-links": "^8.2.9",
-    "@storybook/blocks": "^8.2.9",
-    "@storybook/manager-api": "^8.2.9",
-    "@storybook/test": "^8.2.9",
+    "@storybook/addon-a11y": "^8.3.5",
+    "@storybook/addon-docs": "^8.3.5",
+    "@storybook/addon-essentials": "^8.3.5",
+    "@storybook/addon-links": "^8.3.5",
+    "@storybook/blocks": "^8.3.5",
+    "@storybook/manager-api": "^8.3.5",
+    "@storybook/test": "^8.3.5",
     "@storybook/test-runner": "^0.19.0",
-    "@storybook/theming": "^8.2.9",
-    "@storybook/web-components": "^8.2.9",
-    "@storybook/web-components-vite": "^8.2.9",
+    "@storybook/theming": "^8.3.5",
+    "@storybook/web-components": "^8.3.5",
+    "@storybook/web-components-vite": "^8.3.5",
     "@vitest/ui": "^1.6.0",
-    "axe-playwright": "^2.0.2",
+    "axe-playwright": "^2.0.3",
     "concurrently": "^8.2.2",
     "custom-elements-manifest": "^2.1.0",
-    "eslint": "^9.10.0",
+    "eslint": "^9.12.0",
```

Comment:
## eslint

We have an `eslint` dependency but not a script to use it.

Running `npx eslint` flags browser variables `window` and `document` as being undefined.

I found this was avoidable by installing a `globals` package and adding the global browser config to our `eslint.config.js`

We have #31 which outlines adding the eslint lit plugin. I could add this requirement to this issue or create a new separate issue to track.

Alternatively, we can uninstall `eslint` if we decide to rely on prettier linting instead.